docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -155,35 +181,74 @@ Remember that if an argument for `-Dquarkus.native.additional-build-args` includ
 it needs to be escaped to be processed correcly, e.g. `\\,`.
 ====
 
+== Inspecting Native Executables
+
 Given a native executable, various Linux tools can be used to inspect it.
+To allow support a variety of environments,
+inspections will be done from within a Linux container.
+Let's create a Linux container image that with all the tools required for this guide:
```

Comment:
```suggestion
Let's create a Linux container image with all the tools required for this guide:
```

Reply:
Isn't something missing after the `that`?

Reply:
Good point, I don't remember anything missing that I wanted to say. I was probably simply typing one way to say it and ended up with the other :)